tests/unit/src/main/scala/tests/BuildServerInitializer.scala

```diff
@@ -157,7 +157,6 @@ object SbtServerInitializer extends BuildServerInitializer {
         sbtLaunchJar.toString,
         "xsbt.boot.Boot",
         "-bsp",
-        s"--sbt-launch-jar=$sbtLaunchJar",
```

Comment:
See our current implementation here - https://github.com/sbt/sbt/blob/v1.10.3/protocol/src/main/scala/sbt/internal/bsp/BuildServerConnection.scala

Reply:
Actually, it seems to fail always with `--sbt-launch-jar`, but it's not added automatically with bspConfig command, so I guess this is fine to just remove